docs/application-setup.md

```diff
@@ -207,9 +206,14 @@ or execute these commands manually to get the same result:
 <!-- accordion:start -->
 ### Click to expand
 ```bash
-vue create -d my-vue-app && cd my-vue-app
-npm i vue-router vuex @luigi-project/core @luigi-project/client fundamental-styles @sap-theming/theming-base-content
-npm i -D sass-loader node-sass webpack@4.43.0 webpack-cli@3.3.12 @babel/core @babel/preset-env babel-loader
+# create sample vue app
+vue create -d $folder && cd $folder
+
+# install dependencies
+curl https://raw.githubusercontent.com/SAP/luigi/master/core/examples/luigi-example-vue/package.json > package.json
+npm i
+# as soon as new package.json under vue example released this line can be removed
+npm i webpack-cli@4.2.0 node-sass@4.14.1
 
 mkdir -p src/luigi-config src/assets/scss src/views public/assets
 
```

Comment:
Maybe place the `# remove default index, will be replaced with example assets` comment in a separate line, easier to copy that way